scout/build/disease.py

```diff
@@ -19,7 +19,6 @@ def build_disease_term(disease_info, alias_genes={}):
         disease_obj(dict): Formated for mongodb
 
         disease_term = dict(
-            _id = str, # Same as disease_id
             disease_id = str, # required, like OMIM:600233
```

Comment:
Wait a minute: there was a second field already?

Reply:
They look like this atm:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/28093618/230016697-0cec76d3-d2d1-4fd8-bc78-cb87e41a53a0.png">